modules/deployment-scripts/create-kv-certificate/main.bicep

```diff
@@ -2,7 +2,7 @@
 param akvName string
 
 @description('The location to deploy the resources to')
-param location string = resourceGroup().location
```

Comment:
Is the change to remove the default location value intentional?

Reply:
Ah, yes. Should have provided context for this one.

The Marketplace uses ARM-TTK to automatically validate templates and this causes it to fail one of the checks. The version they use can be found here: aka.ms/arm-ttk-marketplace
This the error: https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/linter-rule-no-hardcoded-location

To get them to override the issue, I have to manually ask for an exception for each of my submissions. This was the main issue blocking me from using a number of our templates before the winter break.

Reply:
I see. In that case, I think we should bump the major version of the module, since updating default value is a breaking change.